wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -164,29 +180,39 @@ sample =
 describe :: WalletError -> String
 describe = \case
     NotEnoughMoney _ ->
-         "Not enough available coins to proceed."
+        "Not enough available coins to proceed."
```

Comment:
If it is really necessary to do whitespace changes it should be done in a separate whitespace only PR and not mixed up with a huge number of other much more important changes.